src/sagemaker/image_uri_config/huggingface.json

```
@@ -139,5 +139,80 @@
                 }
             }
         }
+    },
+
+    "inference": {
+        "processors": ["gpu"],
```

We also have `cpu` inference images.
```suggestion
        "processors": ["cpu", "gpu"],
```